docs/Apps/Metabolomic Data/Labeled LC-MSMS Workflow.md

```diff
@@ -211,7 +211,7 @@ After uploading the necessary files, click on *Calculate Phi*.
 
 - **Absent Phis:** This sub-tab contains a downloadable list of absent Phi values that the app could not match with the identifier table.
 
-- **Identifier Expression:** This sub-tab displays the list of identifier formulas used to calculate Phi values for the generic labeled data. Here in the figure 28, we have for the glutamine label.
+- **Identifier Expression:** This sub-tab displays the list of identifier formulas used to calculate Phi values for the generic labeled data.
 
 ![Identifier Expression](../../img/LabeledLC-MS2Workflow/glutamine_identifier_expressions.png) <center>**Figure 29.** Identifier Expression (generic label)</center>
```

Comment:
If you are removing glutamine label information from the text above. Under figure, description add glutamine label instead of the generic label to give more context on what the figure is depicting

Reply:
@KMaheswari can you comment on this?

Reply:
@Shreya-crypto @avijitzutshi By default the app uses Glutamine as the generic label (in case the user hasn't uploaded any other label data), hence that last line in the paragraph. However, that generic label could be any other label as well apart from Glutamine that the user can use on the app.